sdcm/mgmt/operator.py

```diff
@@ -201,6 +201,27 @@ def set_scylla_cluster(self, scylla_cluster):
     def set_cluster_name(self, name: str):
         self.cluster_name = name
 
+    def are_healthchecks_done(self):
+        # NOTE: operator has 3 healthchecks which we should check:
+        # +---------------------------------+-----------+-------------------------------+--------+
+        # | Task                            | Arguments | Next run                      | Status |
+        # +---------------------------------+-----------+-------------------------------+--------+
+        # | healthcheck/uuid-foo            |           | 06 May 22 12:27:45 UTC (+15s) | DONE   |
+        # | healthcheck_alternator/uuid-bar |           | 06 May 22 12:27:45 UTC (+15s) | DONE   |
+        # | healthcheck_rest/uuid-quuz      |           | 06 May 22 12:28:30 UTC (+1m)  | DONE   |
+        # +---------------------------------+-----------+-------------------------------+--------+
+        healthchecks = self._get_task_list_filtered(prefix="healthcheck", task_class=HealthcheckTask)
+        return all((healthcheck.status == TaskStatus.DONE for healthcheck in healthchecks))
+
+    def wait_for_healthchecks(self):
```

Comment:
isn't this need also for regular manager CLI code ? not only for operator ?

Reply:
Maybe. Let's hear from @ShlomiBalalis

Reply:
We never really had a need for it, but it can't really hurt? Though, I don't understand how waiting for the healthcheck to end alone is enough. If anything, we should check the status of the nodes as well, so we will see that they all are in UN, right?
Otherwise I don't think it helps much

Reply:
We never ran into it in regular manager use, but it can't hurt too I guess

Reply:
Well @ShlomiBalalis if so, you'll refactor it, if it would be needed via the CLI too...